src/client/hypertable/CephBroker.cc

```diff
@@ -409,19 +409,19 @@ int CephBroker::rmdir_recursive(const char *directory) {
 void CephBroker::flush(ResponseCallback *cb, uint32_t fd) {
   OpenFileDataCephPtr fdata;
 
-  HT_DEBUGF("flush fd=%d", fd);
+  HT_DEBUGF("flush fd=%lu", (Lu)fd);
```

Comment:
PRIu32

Reply:
I can do that, but this needs changes in more than these places since this kind of code is already used a lot in the file.